integration-test/integration_test/bot.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+import fire
+import tomlkit
+import os
+from pathlib import Path
+import subprocess
+import json
+
+class CLI:
+    def prepare(self, tendermint=None, tmkms=None, tmhome=None, kmsconfig=None):
```

Comment:
```suggestion
    def prepare(self, tendermint="tendermint", tmkms="tmkms-softsign", tmhome=".tendermint", kmsconfig="tmkms.toml"):
```
I think you can set the defaults here directly.